applications/tari_dan_wallet_cli/src/command/account.rs

```diff
@@ -90,8 +96,23 @@ pub struct ClaimBurnArgs {
     /// Optional proof JSON from the L1 console wallet. If not provided, you will be prompted to enter it.
     #[clap(long, short = 'j', alias = "json")]
     proof_json: Option<serde_json::Value>,
-    #[clap(long, short = 'f')]
-    fee: Option<u64>,
+    #[clap(long, short = 'f', default_value_t = 1000)]
+    fee: u64,
+}
+
+#[derive(Debug, Args, Clone)]
+pub struct RevealFundsArgs {
+    /// The account name where the funds will be revealed
+    account_name: String,
```

Comment:
```suggestion
   #[clap(long, short='n', alias='name')]
    account_name: String,
```

Reply:
It's a mandatory field for now, so feels better to me to have it as an arg. `reveal <from_acc_name> <amount>`. If we maybe added a default account concept (maybe default account to pay fees), we could use that if not specified. Then I'd change it to a flag. 

Happy to change it if you disagree.

Reply:
nah, leave it as is for now